### ISSUE TITLE
Change btap_results measure test to allow use of existing osm file.

### ISSUE THREAD

**Issue by ckirney** [canmet-energy/btap_tasks#217]
### Description
Adjust the test.rb file for the btap_results (costing measure) to allow use of an existing .osm file rather than having to run openstudio-standards to make one.

### Approach
Add a variable which, if empty, will run test.rb as normal.  If not empty it will run with the file in the array.  Also copy the openstudio-stardards necb regression test files to btap_costing to use as existing osm files.

### Testing Plan
If it runs it works.

### Waiting On
Someone's time.

### Repositories Involved

https://github.com/canmet-energy/btap_costing
https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|PERD   |P-002518.001.03	|Building Archetype	|Phylroy Lopez

**Comment by ckirney** [comment]
Can now use test.rb in the btap_results measure in one of three ways:
1.  Set 'use_existing_osm' to false to use current default functionality (btap_results will have openstudio-standards create an osm based on what you set in building_type, template, and epw_file)
2.  Set 'use_exsisting_osm = true' and leave 'custom_file = nil' to use an existing openstudio-standards necb regression osm file based on the building_type, epw_file (only Calgary or Kuujuaq), and 'template' settings
3.  Set 'use_existing_osm = true' and set 'custom_file' to the name of an existing osm file to use that osm for the tests (note you can set the directory the file is in by changing @test_fold to the directory starting from /btap_costing/).

**Comment by ckirney** [comment]
How to use test.rb in the btap_results measure:

  1.  Current default:  Have the test create the file and run a full simulation before testing the measure.
       To Use:       
       -Set 'use_existing_osm' to false
       -Set 'building_type' to the building archetype you want to test
       -Set 'epw_file' to the weather file you want to use
       -Set 'template' to the NECB version you want to use
       -Set '@test_output' = nil
       -Set 'custom_file' = nil

  2.  Use one of the openstudio-standards regression files in '/os_standards_reg_tests' but still conduct a simulation.
       To Use:
       -Set 'use_existing_osm' to true
       -Set 'building_type' to the building archetype you want to test
       -Set 'epw_file' to the weather file either Calgary or Kuujjuaq
       -Set 'template' to the NECB version you want to use
       -Set '@test_output' = nil
       -Set 'custom_file' = nil
       -Set '@test_fold' = "/os_standards_reg_tests/"

  3.  Use your own existing osm file but still have OpenStudio run a simulation.
       To Use:
       -Set 'use_existing_osm' to true
       -Set 'epw_file' to the weather file used in the osm.
       -Set 'template' to the NECB version in the osm.
       -Set '@test_output' = nil
       -Set 'custom_file' = the name of your osm
       -Set '@test_fold' = the folder your osm is in assuming you are starting from the btap_costing repository folder location.  Make sure to add '/' at the start and end of the directory tree.

  4.  Use your own existing osm file with which you have already done a simulation.
       To Use:
       -Set 'use_existing_osm' to true
       -Set 'epw_file' to the weather file used in the osm.
       -Set 'template' to the NECB version in the osm.
       -Set '@test_output' = The path to the simulation results folder.  This folder should have the default OpenStudio results structure and nomenclature.  The path assumes you are starting in the btap_costing repository folder.  Do NOT include starting and ending '/'.
       -Set 'custom_file' = the name of your osm
       -Set '@test_fold' = the folder your osm is in assuming you are starting from the btap_costing repository folder location.  Make sure to add '/' at the start and end of the directory tree.
